packages/create-ponder/src/subgraph.ts

```diff
@@ -99,6 +100,21 @@ export const fromSubgraphId = async ({
     }),
   );
 
+  // Copy over the schema.graphql file.
+  const schemaRaw = await provider.fetchIpfs(
+    manifest.schema.file["/"].slice(6),
+  );
+
+  const schemaPath = path.join(rootDir, "./ponder.schema.ts");
+
+  // Translate and write the schema file
+  const translatedSchema = translateSchema(schemaRaw);
+
+  writeFileSync(
+    schemaPath,
+    await prettier.format(translatedSchema, { parser: "typescript" }),
```

Comment:
FWIW, this will take care of whitespace and newline characters, so we can omit them above if it helps readability.

Reply:
Yee I agree